src/HomePage/components/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { FunctionComponent } from 'react';
+import './NavBar.css'
+
+export const NavBar: FunctionComponent<{categories: Array<string>}> = ({ categories }) => {
+    return <div>
+        {categories.map((category: any) => <><span className='movie-category'>{category}</span></>)}
```

Comment:
for this specific case you don't need react fragement here, because it wraps just one element. it is needed when you need to pass several elements.